custom_components/amplipi/media_player.py

```diff
@@ -283,19 +512,30 @@ async def async_play_media(self, media_type, media_id, **kwargs):
 
     async def async_select_source(self, source):
         if self._source is not None and self._source.name == source:
-            await self._update_source(SourceUpdate(
-                input='local'
-            ))
+            await self._client.set_source(
+                self._id,
+                SourceUpdate(
+                    input='local'
```

Comment:
input=local is deprecated in the set_source api, IIRC it is dynamically replaced with input="steam={996+source_id}". We should use the newer convention here.